### ISSUE TITLE
Implementing Commands

### ISSUE THREAD

**Issue by Chris-Prime** [Chris-Prime/FactionsPE#4]
- [x] Create
- [x] Leave
- [ ] Invite
- [ ] Join
- [ ] Relation
- [ ] Claim
- [ ] Map
- [ ] Chat
- [ ] Override (bypass)
- [ ] Kick
- [ ] Promote
- [ ] Demote

**Comment by ArrowDCA** [comment]
will their be warzone claims?

**Comment by Chris-Prime** [comment]
Yes https://github.com/Chris-Prime/FactionsPE/blob/reborn/src/factions/manager/Factions.php#L81-L91

**Comment by ArrowDCA** [comment]
nices
